src/components/OperationsList/index.js

```diff
@@ -240,43 +272,40 @@ class OperationsList extends Component<Props> {
 
     return (
       <Defer>
-        <Card flow={1} title={title} p={0}>
-          <Box horizontal pt={4}>
-            <HeaderCol size={DATE_COL_SIZE}>{t('operationsList:date')}</HeaderCol>
-            {withAccount && (
-              <HeaderCol size={ACCOUNT_COL_SIZE}>{t('operationsList:account')}</HeaderCol>
-            )}
-            <HeaderCol grow>{t('operationsList:address')}</HeaderCol>
-            <HeaderCol size={AMOUNT_COL_SIZE} justifyContent="flex-end">
-              {t('operationsList:amount')}
-            </HeaderCol>
-            <HeaderCol size={CONFIRMATION_COL_SIZE} px={0} />
-          </Box>
-
-          <Box>
-            {operations.map(tx => (
-              <Operation
-                account={account}
-                t={t}
-                key={`{${tx.hash}${tx.account ? `-${tx.account.id}` : ''}`}
-                withAccount={withAccount}
-                onAccountClick={onAccountClick}
-                minConfirmations={minConfirmations}
-                tx={tx}
-              />
-            ))}
-          </Box>
+        <Box>
+          <Card flow={1} title={title} p={0}>
+            <Box>
+              {operations.map(tx => {
+                const acc = account || tx.account
```

Comment:
what is this `account || tx.account` ? 🤔

Reply:
You can have `account` in 2 ways, so, in operation object (for example, in Dashboard https://github.com/LedgerHQ/ledger-wallet-desktop/blob/master/src/components/DashboardPage/index.js#L67 when we want to display OperationsList with different accounts) and with props (in AccountPage (https://github.com/LedgerHQ/ledger-wallet-desktop/blob/master/src/components/AccountPage/index.js#L160 we pass directly props)

Reply:
oh yeah I see. makes sense. I guess later we will have to change that if tx.account is gone. like `account || findInAccounts(accounts, tx.accountId)`,